app/bundles/CoreBundle/Test/MauticMysqlTestCase.php

```diff
@@ -114,13 +127,16 @@ protected function truncateTables(string ...$tables): void
     private function applySqlFromFile($file)
     {
         $connection = $this->connection;
-        $password   = ($connection->getPassword()) ? " -p{$connection->getPassword()}" : '';
-        $command    = "mysql -h{$connection->getHost()} -P{$connection->getPort()} -u{$connection->getUsername()}$password {$connection->getDatabase()} < {$file} 2>&1 | grep -v \"Using a password\" || true";
+        $command    = "mysql -h{$connection->getHost()} -P{$connection->getPort()} -u{$connection->getUsername()} {$connection->getDatabase()} < {$file}";
```

Comment:
Because you are using more advanced `symfony/process` can you please change whole assembly of the command line with this library? This will bring proper escaping at least.

Reply:
@biozshock 

indeed, that's an option now, a reworked it to use a "prepared command"